ros/src/tl_detector/light_classification/tl_classifier.py

```diff
@@ -35,13 +55,28 @@ def simple_opencv_red_color_classifier(self,image):
 
         return TrafficLight.UNKNOWN
 
-    def dl_based_classifier(self,image):
+    def dl_based_classifier(self, image):
+        image_np = np.expand_dims(np.asarray(image, dtype=np.uint8), 0)
+                     
+        # Actual detection
+        (boxes, scores, classes) = self.sess.run([detection_boxes, detection_scores, detection_classes], feed_dict={image_tensor: image_np})
+
+        # Remove unnecessary dimensions
+        scores = np.squeeze(scores)
+        classes = np.squeeze(classes)
+        
+        for i in range(len(classes)):
+		print('class=', classes[i])
+		print('score=', scores[i])
+            	if classes[i] == 2 and scores[i] > 0.5: # if red light with confidence more than 50%
```

Comment:
Suggestion:
Also return the Yellow light, (although we ignore the yellow light for now, might be useful later)

Reply:
can you print the name of the classes? i.e. 0 == green, 1 == yellow 2 == red, 3 == unknown?

Reply:
I think we should treat yellow light as red. When yellow follows green, the driver should start decelerating. When yellow follows red, the driver should prepare to accelerate, but not start accelerating until there is a green light. For an unsophisticated self-driving car, yellow essentially means red.